src/Files.App/UserControls/SidebarControl.xaml.cs

```diff
@@ -214,6 +214,11 @@ private List<ContextMenuFlyoutItemViewModel> GetLocationItemMenuItems(INavigatio
 				{
 					IsVisible = options.ShowEmptyRecycleBin,
 				}.Build(),
+				new ContextMenuFlyoutItemViewModelBuilder(commands.RestoreAllRecycleBin)
+				{
+					IsVisible = options.ShowEmptyRecycleBin
+								&& RecycleBinHelpers.RecycleBinHasItems(),
```

Comment:
`Empty recycle bin` is shown in the context menu (but is disabled) if the recycle bin is empty. I think we should do the same with this action

Reply:
Done.